sys/auto_init/auto_init.c

```diff
@@ -304,4 +309,12 @@ void auto_init(void)
     DEBUG("Auto init UDP module.\n");
     ng_udp_init();
 #endif
+#ifdef MODULE_NG_NETIF
+    DEBUG("Auto init netif module.\n");
+    ng_netif_init();
```

Comment:
(`ng_netif_init()` just zeros the `netif` array, which is zero anyway on startup)